files/image_config/ntp/mgmtvrf_ntpstat

```diff
@@ -0,0 +1,5 @@
+ip link set dev lo master mgmt
```

Comment:
same question as above.

Reply:
Yes, we shall create a new "lo0" (or "lo1") interface and use it for this purpose.